.github/workflows/wheels.yaml

```diff
@@ -70,6 +70,13 @@ jobs:
     name: Publish wheels to PyPI
     runs-on: ubuntu-latest
     if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+
+    environment:
+      name: pypi
```

Comment:
But our repository doesn't have `pypi` environment?

Do we even need it? Isn't that empty in `si4ti` too?